.github/workflows/ci.yml

```diff
@@ -109,19 +95,22 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
+      - name: "Services: Start Test Services"
+        uses: hoverkraft-tech/compose-action@v2.2.0
+        with:
+          compose-file: "./docker/compose-ci.yaml"
+
+      - name: Build everything
```

Comment:
We shouldn't need to build everything here, but it's the easiest way to make sure all the dependencies of the tests are build.

turborepo will clean this up.